src/screencast/screencast.c

```diff
@@ -585,12 +585,73 @@ static int method_screencast_start(sd_bus_message *msg, void *data,
 	if (ret < 0) {
 		return ret;
 	}
-	ret = sd_bus_message_append(reply, "{sv}",
```

Comment:
Did you have any specific reason why you replaced this with multiple calls? Was it for conditional properties, or did you like the explicit version better, or something else?

Reply:
Just for conditional properties. If you know a better way to do this I'm happy to change it.